raiden/event_handler.py

```diff
@@ -245,6 +245,15 @@ def handle_channelnew(self, state_change):
                 token_address,
                 channel_address,
             )
+            # This is little hacky, used when bootstraping a token network.
+            # See ConnectionManager for details.
+            have_bootstrap = ((participant1 == connection_manager.BOOTSTRAP_ADDR) or
+                              (participant2 == connection_manager.BOOTSTRAP_ADDR))
+            if not have_bootstrap:
+                self.raiden.start_health_check_for(
+                    participant2 if
+                    participant1 == self.raiden.address else
+                    participant1)
```

Comment:
Humm, two things:
1. This check is really noisy.
2. I don't think we should register a netting channel with the bootstrap address, this was an oversight.

Suggestion, simplify the check and move it above line `243`, e.g.:
```
is_participant = self.raiden.address in (participant1, participant2)
is_bootstrap = connection_manager.BOOTSTRAP_ADDR in (participant1, participant2)
if is_participant and not is_bootstrap:
```